### PR TITLE
Ericthompson/md docs updates

### PR DESCRIPTION
**Pull request by ericthompson**
This PR adds md docs, and cleans up our existing ones to reflect all of the changes in the project structure and the new, screenshot-including component doc files.

### CODE REVIEW

File: ghdocs/CONTRIBUTING.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 Office UI Fabric JS is an evolving snapshot of the Office 365 Design Language's components, provided by the OneDrive and SharePoint Design Studio.
 
-Currently, this set of components is in a **paused** state by the Fabric team while we work on the other control sets, but is still open to contributions that help improve the stability or usability of the components.
+Currently, teh Fabric team is focusing on other control sets, but we are still open to contributions that help improve the stability or usability of the components.
```
Comment (Jahnp) [inline ghdocs/CONTRIBUTING.md]: Mispelling: teh

**Comment by Jahnp** [comment]
Approved :coffee:

File: ghdocs/CONTRIBUTING.md (hunk 1 above)
Comment (ericthompson) [inline ghdocs/CONTRIBUTING.md]: Fixed - good catch!

**Comment by mikewheaton** [comment]
Approved ❤️

**Pull request merged**